### PR TITLE
Rebranding

### PR DESCRIPTION
**Pull request by miiu96**
- Rebranding and update dependencies

### CODE REVIEW

File: trieTools/trieToolsCommon/vars.go [hunk 1]
```diff
@@ -1,10 +1,10 @@
 package trieToolsCommon
 
 import (
-	"github.com/ElrondNetwork/elrond-go-core/hashing/blake2b"
-	"github.com/ElrondNetwork/elrond-go-core/marshal"
-	"github.com/ElrondNetwork/elrond-go-storage/storageUnit"
-	elrondConfig "github.com/ElrondNetwork/elrond-go/config"
+	"github.com/multiversx/mx-chain-core-go/hashing/blake2b"
+	"github.com/multiversx/mx-chain-core-go/marshal"
+	elrondConfig "github.com/multiversx/mx-chain-go/config"
```
Comment (BeniaminDrasovean) [inline trieTools/trieToolsCommon/vars.go:6]: elrondConfig

File: trieTools/balancesExporter/trie/config.go [hunk 2]
```diff
@@ -1,8 +1,8 @@
 package trie
 
 import (
-	"github.com/ElrondNetwork/elrond-go-storage/storageUnit"
-	elrondConfig "github.com/ElrondNetwork/elrond-go/config"
+	elrondConfig "github.com/multiversx/mx-chain-go/config"
```
Comment (BeniaminDrasovean) [inline trieTools/balancesExporter/trie/config.go:4]: elrondConfig

File: trieTools/trieToolsCommon/common.go [hunk 3]
```diff
@@ -7,28 +7,28 @@ import (
...
-	"github.com/ElrondNetwork/elrond-go/testscommon"
-	"github.com/ElrondNetwork/elrond-go/trie"
-	hashesHolder "github.com/ElrondNetwork/elrond-go/trie/hashesHolder/disabled"
-	"github.com/ElrondNetwork/elrond-tools-go/trieTools/trieToolsCommon/components"
+	"github.com/multiversx/mx-chain-core-go/core/check"
+	"github.com/multiversx/mx-chain-core-go/core/pubkeyConverter"
+	nodeFactory "github.com/multiversx/mx-chain-go/cmd/node/factory"
+	"github.com/multiversx/mx-chain-go/common"
+	commonDisabled "github.com/multiversx/mx-chain-go/common/disabled"
+	elrondConfig "github.com/multiversx/mx-chain-go/config"
```
Comment (BeniaminDrasovean) [inline trieTools/trieToolsCommon/common.go:15]: elrondConfig

File: trieTools/trieToolsCommon/vars.go (hunk 1 above)
Comment (miiu96) [reply trieTools/trieToolsCommon/vars.go:6]: fixed.

File: trieTools/balancesExporter/trie/config.go (hunk 2 above)
Comment (miiu96) [reply trieTools/balancesExporter/trie/config.go:4]: elrondConfig

File: trieTools/trieToolsCommon/common.go (hunk 3 above)
Comment (miiu96) [reply trieTools/trieToolsCommon/common.go:15]: fixed

File: .github/workflows/pr-tests.yml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+        id: go
+
+      - name: Check out code
+        uses: actions/checkout@v2
+      - name: Get dependencies
+        run: |
+          make get-dependencies
+      - name: Unit tests
+        run: |
+          make test
```
Comment (BeniaminDrasovean) [inline .github/workflows/pr-tests.yml:27]: Add empty line at the end of the file

File: Makefile [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+	cd dbMerger && go get -v -t -d ./...
+	cd elasticreindexer && go get -v -t -d ./...
+	cd tokensRemover && go get -v -t -d ./...
+	cd trieTools && go get -v -t -d ./...
+
+test: |
+	cd dbMerger && go test ./...
+	cd elasticreindexer && go test ./...
+	cd tokensRemover && go test ./...
+	cd trieTools && go test ./...
```
Comment (BeniaminDrasovean) [inline Makefile:11]: Add empty line at the end of the file

**Review by BeniaminDrasovean — APPROVED** [review]

File: .github/workflows/pr-tests.yml [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+jobs:
+  test:
+    name: Unit
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v2
+        with:
+          go-version: ^1.17.6
```
Comment (bogdan-rosianu) [inline .github/workflows/pr-tests.yml:17]: remove `^` - it can fetch go 1.19 which is not currently supported by the node

**Review by bogdan-rosianu — APPROVED** [review]

File: .github/workflows/pr-tests.yml (hunk 4 above)
Comment (miiu96) [reply .github/workflows/pr-tests.yml]: added

File: Makefile (hunk 5 above)
Comment (miiu96) [reply Makefile]: added

File: .github/workflows/pr-tests.yml (hunk 6 above)
Comment (miiu96) [reply .github/workflows/pr-tests.yml]: changed

**Review by BeniaminDrasovean — APPROVED** [review]

**Pull request merged**